ikarus/finiteelements/mechanics/kirchhoffloveshell.hh

```diff
@@ -305,6 +305,8 @@ namespace Ikarus {
         }
       }
       K.template triangularView<Eigen::StrictlyLower>() = K.transpose();
+      VolumeType::calculateMatrix(par, K, dx);
```

Comment:
Maybe add a comment here about follower loads. Only if you want.

Reply:
I will add a short comment for future reference